table/block_based/block_based_table_reader.cc

```diff
@@ -1715,6 +1715,9 @@ void BlockBasedTable::RetrieveMultipleBlocks(
       req_offset_for_block.emplace_back(0);
     }
     req_idx_for_block.emplace_back(read_reqs.size());
+
+    PERF_COUNTER_ADD(block_read_count, 1);
+    PERF_COUNTER_ADD(block_read_byte, block_size(handle));
```

Comment:
Putting the code here (as opposed to later) is the number of reads we attempted.  Is that what is expected or should it be put later and be the number of reads that were successful?

Reply:
Its consistent with how the counters are updated in block_fetcher.cc. Its not explicitly documented in perf_context.h, so just retain existing behavior.